boom/osprofile.py

```
@@ -720,8 +723,8 @@ def _from_file(self, profile_file: str):
         try:
             # Call subclass _from_data() hook for initialisation
             self._from_data(profile_data, dirty=False)
-        except ValueError as e:
-            raise ValueError(str(e) + f"in {profile_file}")
+        except ValueError as err:
+            raise ValueError(str(err) + f"in {profile_file}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)